src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -324,9 +368,31 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 		profile.setOptionValue('applicationName', applicationName);
 		profile.setOptionValue('databaseDisplayName', profile.databaseName);
 		profile.setOptionValue('groupId', profile.groupId);
+		// Set all advanced options
+		let advancedOptions = this.getAdvancedOptions(args.connectionProperties, profile.getOptionKeyIdNames());
```

Comment:
Make sure to check that this is already URI-decoding the content before it gets here (for example have options content that contains escaped reserved characters like & or ?)